<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDialog</class>
 <widget class="QDialog" name="ClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client x</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Client Information</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_client_ip">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>256:256:256:256</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_client_port">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>7777</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Not connected</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBx_chat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>271</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Chat</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Message to server:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_msg_to_server">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_send_msg">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-3</number>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Message from server:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_msg_from_server">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>261</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pBtn_close">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gBx_server_connnect">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Server Connect</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_server_ip">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_server_port">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_connect_to_server">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect to server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBtn_disconnect_from_server">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect from server</string>
    </property>
   </widget>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
